recipes/dicom2nifti/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dicom2nifti" %}
+{% set version = "2.2.1" %}
+{% set sha256 = "b712e5ee2c367d12cd0a3139bcd604daf1da454bcc8da8b204882696c8c39b77" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/icometrix/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
```

Comment:
I don't think `setuptools` is needed here.

Reply:
Thanks for the review. The setuptools was removed.